examples/advanced_agents/sandbox_agent/src/nat_sandbox_agent/sandbox/daytona_sandbox.py

```
@@ -0,0 +1,228 @@
+        except Exception as e:
+            logger.exception("Command execution failed")
+            return CommandResult(
+                exit_code=-1,
+                stdout="",
+                stderr=str(e),
+            )
+
+    async def read_file(self, path: str) -> str:
+        """Read file content from the Daytona sandbox."""
+        if not self._sandbox:
+            raise RuntimeError("Sandbox not started")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)